File: LuaLibs/src/MyLua/Libs/EventHandle/ResEventDispatch.lua
local M = GlobalNS.Class(GlobalNS.EventDispatch)
M.clsName = "ResEventDispatch"
GlobalNS[M.clsName] = M

function M:ctor()
    
end

function M:dtor()

end

function M:dispatchEvent(dispatchObject)
    self.super.dispatchEvent(self, dispatchObject)
    self.clearEventHandle();
end

return M

File: PyLibs/src/Libs/EventHandle/ResEventDispatch.py
# -*- coding: utf-8 -*-

from Libs.EventHandle.EventDispatch import EventDispatch

class ResEventDispatch(EventDispatch):
    def __init__(self):
        super(ResEventDispatch, self).__init__()
        
        self.mTypeId = "ResEventDispatch";


    def dispatchEvent(self, dispatchObject):
        super(ResEventDispatch, self).dispatchEvent(dispatchObject);
        self.clearEventHandle();


